yellowbrick/model_selection/__init__.py

```diff
@@ -16,3 +16,4 @@
 
 from .learning_curve import LearningCurve, learning_curve
 from .validation_curve import ValidationCurve, validation_curve
+from .cv import CVScores, cv
```

Comment:
Thanks for including these imports at the top level! I think this is where the tests are failing, this should be:

```python
from .cv import CVScores, cv_scores
```